parent/pom.xml

```
@@ -474,6 +474,16 @@
                                 </dependency>
                             </dependencies>
                         </plugin>
+                        <plugin>
+                            <groupId>org.apache.maven.plugins</groupId>
+                            <artifactId>maven-surefire-plugin</artifactId>
+                            <configuration>
+                                <!-- Groovy tests access private fields -->
+                                <argLine>
+                                    --add-opens=java.base/java.util=ALL-UNNAMED
```

```suggestion
                                    --add-opens=java.base/java.util=ALL-UNNAMED
                                    --add-opens=java.base/java.lang=ALL-UNNAMED
```

We may want to also add java.lang, and there might be a few other things groovy is making use of.

We can either try to add the most common ones here, or add a args to the individual modules as needed